core/src/main/java/pl/agh/edu/actor/utils/LanguageLabel.java

```diff
@@ -0,0 +1,34 @@
+package pl.agh.edu.actor.utils;
+
+import com.badlogic.gdx.graphics.g2d.BitmapFont;
+import com.badlogic.gdx.scenes.scene2d.utils.NinePatchDrawable;
+
+import pl.agh.edu.actor.GameSkin;
+import pl.agh.edu.language.LanguageChangeListener;
+import pl.agh.edu.language.LanguageManager;
+
+public class LanguageLabel extends CustomLabel implements LanguageChangeListener {
+	private final String languagePath;
+
+	public LanguageLabel(String languagePath, String font) {
+		super(font);
+		this.languagePath = languagePath;
+		LanguageManager.addListener(this);
+		onLanguageChange();
+	}
+
+	@Override
+	public void onLanguageChange() {
+		this.setText(LanguageManager.get(languagePath));
+	}
+
+	public void setUnderscore() {
+		LabelStyle labelStyle = new LabelStyle(getStyle());
+		NinePatchDrawable underscore = new NinePatchDrawable(GameSkin.getInstance().getPatch("underscore"));
+
+		// underscore = underscore.tint(SkinColor(SECONDARY, _500));
+
+		labelStyle.background = underscore;
```

Comment:
nie jestem pewnien czy ustalanie backgroundu jako underscore to jest dobry pomysł bo np w takim razie nie będziemy mogli ustawić innego backgroundu czyli konien końców w takim case albo możemy mieć labelkę z underscorem albo z backgroundem

Reply:
If you want to apply an underline to a font, a great technique for that is to set an underline 9patch to the background.

Reply:
w takim razie bez podkreslenia albo komplkujemy sprawe i działamy na stacku

Reply:
a a metodka draw w odpowiednim miejscu nie styknie?